<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>841</width>
     <height>591</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_Distancia">
    <attribute name="title">
     <string>Distancia</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>491</width>
       <height>461</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap>imagen2.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_Tinaco">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>221</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(76, 178, 255);
selection-color: rgb(255, 205, 205);</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_Llenado">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_TuboSalida">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>distancia.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_Litros">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>90</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label</zorder>
    <zorder>progressBar_Tinaco</zorder>
    <zorder>progressBar_Llenado</zorder>
    <zorder>progressBar_TuboSalida</zorder>
    <zorder>lcdNumber_Litros</zorder>
   </widget>
   <widget class="QWidget" name="tab_Humedad">
    <attribute name="title">
     <string>Humedad</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>841</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Humedad.webp</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>731</width>
       <height>541</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>med_humedad.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_Humedad">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>280</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Temperatura">
    <attribute name="title">
     <string>Temperatura</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>0</y>
       <width>951</width>
       <height>571</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>calor.webp</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>401</width>
       <height>561</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>giphy.gif</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>471</width>
       <height>541</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>med_radiacion.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>150</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>med_radiacion.jpg</pixmap>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_Temperatura">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>691</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Fondo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
